static/js/theme-map/Maps/Providers/Leaflet.js

```diff
@@ -119,7 +119,7 @@ class LeafletPin extends ProviderPin {
 
 // Load Function
 
-const yextAPIKey = 'pk.eyJ1IjoieWV4dCIsImEiOiJqNzVybUhnIn0.hTOO5A1yqfpN42-_z_GuLw';
+const yextAPIKey = process.env.MAPBOX_KEY;
```

Comment:
should this be a Leaflet key, not Mapbox?

Reply:
It was hardcoded as the mapbox key, so I replaced it with that.